WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostPagerActivity.java

```diff
@@ -167,6 +173,17 @@ protected void onPause() {
     @Override
     public boolean onOptionsItemSelected(final MenuItem item) {
         if (item.getItemId() == android.R.id.home) {
+            if (mIsFeed) {
```

Comment:
Good point and observation. I should have elaborated on this in the PR description.

> When the user hits the back button while viewing a feed item, this shows the feed preview.

Technically, this happens when hitting the _UP_ button. Yeah, it _looks_ like a back arrow but it's supposed to be a hierarchical navigation. OTOH, tapping on the _back_ button (from the bottom Back/Home/AppSwitcher row) returns to the launching app since that works chronologically. It's this pattern from the docs: https://developer.android.com/training/design-navigation/ancestral-temporal.html

> it's quite unexpected to be returned to a view you didn't start from.

I struggled a bit for this commit to be honest (notice that this change is in its own commit).
Yes, it tends to feel more natural to open the post and return to where you were before (other app) but, it's not clear yet what will be the flow we want to show the user into. For example, I can imagine us wanting the user to get into the Reader and then be able to stay there and browse around a bit more.

The current implementation (UP for feed preview, Back for returning to other app) allows for the quick open of Reader links and back (say, from a digest email) but also gives the chance to browse the feed.

I'm not pushing hard for this one though. It's effectively a proposal even though it tries to follow the docs as well. And since it's not strictly a code issue, we might want to bring more people to chime in as well. Happy to remove it for now if you prefer that. Lemme know what you think. Thanks!